Swift/BRCrypto/common/BRSupport.swift

```diff
@@ -23,6 +23,10 @@ public func asUTF8String (_ chars: UnsafePointer<CChar>) -> String {
     return String (cString: chars, encoding: .utf8)!
 }
 
+public func asUTF8String (_ chars: UnsafePointer<UInt8>) -> String {
```

Comment:
What is this?

Reply:
Bad leftover code from when I was trying to get the message out of the struct. Whoops! Good spot, will remove.